.github/workflows/pull-request-opened-notify-slack.yml

```diff
@@ -0,0 +1,11 @@
+name: Send a Slack notification when PRs are opened
```

Comment:
could add this is to alert someone for review

Reply:
I'm not sure what you mean here. Do you mean adding a ping to the Slack message itself?

Reply:
Changing this to say "Send a Slack notification when PRs are opened to alert of that a review is needed". We care and we want people know that's why this is here so they're not suspicious about why there is a slack ping.

Reply:
Oh gotcha, makes sense! I pushed a commit that changes that text